subtests/docker_cli/kill_stress/kill_stress.py

```diff
@@ -51,7 +51,7 @@ def _map_sequence_to_signals(sequence):
                  signals_set = set of signals, which should be present in the
                                output.
         """
-        stopped = False
```

Comment:
R0204: 71,20: stress._map_sequence_to_signals: Redefinition of stopped type from bool to set